index.js

```diff
@@ -104,8 +104,6 @@ module.exports = function(S) {
 
       return functions.forEach(function(fun) {
         if( -1 !== fun.getRuntime().getName().indexOf('nodejs') ) {
-          // Override s-function.json defined environment!
-          Object.getPrototypeOf( fun.getRuntime() ).getEnvVars = ()=> BbPromise.resolve( {} );
```

Comment:
@marcosnils let me put it opposite way: I wish I could easily specify my own env when locally running functions in SLS, unfortunately I have to properly configure stage variables in a different way than I'd deploy to AWS.

IMO it's useful to mimic sls behavior as close as possible and it's also useful to be able to alter it slightly. Then having a switch between these two would make everyone happy, I suppose. If you agree, then the question is shall I do it or would you?